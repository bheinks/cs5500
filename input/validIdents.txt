Valid_idents		
_1 _ __		_a   __Underscores__		
AaAaAa_Aa	testing1_
and_array_begin_boolean_char_div_do_else_end_false
ifintegernotoforprocedureprogramreadthen
_a_314bc__d
Array And Begin Boolean CHAR DIV Do ELSE END fALSE




