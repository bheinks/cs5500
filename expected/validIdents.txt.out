TOKEN: T_IDENT     LEXEME: Valid_idents
TOKEN: T_IDENT     LEXEME: _1
TOKEN: T_IDENT     LEXEME: _
TOKEN: T_IDENT     LEXEME: __
TOKEN: T_IDENT     LEXEME: _a
TOKEN: T_IDENT     LEXEME: __Underscores__
TOKEN: T_IDENT     LEXEME: AaAaAa_Aa
TOKEN: T_IDENT     LEXEME: testing1_
TOKEN: T_IDENT     LEXEME: and_array_begin_boolean_char_div_do_else_end_false
TOKEN: T_IDENT     LEXEME: ifintegernotoforprocedureprogramreadthen
TOKEN: T_IDENT     LEXEME: _a_314bc__d
TOKEN: T_IDENT     LEXEME: Array
TOKEN: T_IDENT     LEXEME: And
TOKEN: T_IDENT     LEXEME: Begin
TOKEN: T_IDENT     LEXEME: Boolean
TOKEN: T_IDENT     LEXEME: CHAR
TOKEN: T_IDENT     LEXEME: DIV
TOKEN: T_IDENT     LEXEME: Do
TOKEN: T_IDENT     LEXEME: ELSE
TOKEN: T_IDENT     LEXEME: END
TOKEN: T_IDENT     LEXEME: fALSE
